### ISSUE TITLE
rsync_options

### ISSUE THREAD

**Issue by Dan-Essin** [billw2/rpi-clone#143]
I'm now running a clone operation. It is an understatement to say "(can take a long time)".It has now been running for over 10 hours.

I suggest that the rsync_options line:
rsync_options="--force -rltWDEHXAgoptx"

be modified to include --progress:
rsync_options="--force --progress -rltWDEHXAgoptx"

It would be nice to have some idea whether the clone is still actually copying or if the process is stalled for some reason.

**Comment by framps** [comment]
I agree. That's a useful rsync option when you start to use rpi-clone. But later on it should be removed because it will slow down the backup. Said this an additional rpi-clone option which adds this rsync option would be useful.
